content/docs/command-reference/exp/show.md

```diff
@@ -111,6 +125,9 @@ metric or param.
   sorting. This only affects the ordering of experiments derived from the same
   parent commit. Parent commits are always sorted chronologically.
 
+  When combined with the `--pcp` flag, the values of the column `name` will be
+  used to determine the color of the lines in the Parallel Coordinates Plot.
```

Comment:
Misc. copy edit
```suggestion
  > Note that the sort order will determine the color of the lines in `--pcp` mode.
```
Not sure this note is needed though, it'll be evident upon usage.